### ISSUE TITLE
springcloud with alibaba-nacos always get "get upstream ip error"

### ISSUE THREAD

**Issue by slackwareer** [apache/shenyu#5405]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

SpringCloud driven by alibaba nacos always get "get upstream ip error" error.

I traced the code of shenyu, and find a possible bug.
The `NacosServiceInstance` hasn't implemented the `ServiceInstance.getScheme` method,so the `LoggingServerHttpResponse.getUpstreamIpFromHttpDomain` always get the error mentioned above.

### Expected Behavior

It should use the http as its scheme,and the `get upstream ip error` should be disappeared.

### Steps To Reproduce

* Build a bootstrap project with SpringCloud and alibaba-nacos
* Client register a springcloud service to shenyu admin
* visit the service through shenyu
![image](https://github.com/apache/shenyu/assets/239952/929bb9d5-9021-4eb7-85f1-e86a15a86d25)

### Environment

```markdown
ShenYu version(s):2.6.0
nacos-discovery.version:2021.0.1.0
springboot:2.7.17
```

### Debug logs

2024-01-17 05:46:34 [reactor-http-epoll-3] ERROR org.apache.shenyu.plugin.logging.common.body.LoggingServerHttpResponse - get upstream ip error

**Comment by moremind** [comment]
scheme is error

**Comment by slackwareer** [comment]
> scheme is error

Yeah,maybe it should be fixed in Nacos.

**Comment by moremind** [comment]
> > scheme is error
> 
> Yeah,maybe it should be fixed in Nacos.

yeap, I have fix it in https://github.com/alibaba/spring-cloud-alibaba/pull/3545

**Comment by slackwareer** [comment]
> > > scheme is error
> > 
> > 
> > Yeah,maybe it should be fixed in Nacos.
> 
> yeap, I have fix it in [alibaba/spring-cloud-alibaba#3545](https://github.com/alibaba/spring-cloud-alibaba/pull/3545)

good job , waiting the bug fixed version of nacos used in shenyu

**Comment by moremind** [comment]
> > > > scheme is error
> > > 
> > > 
> > > Yeah,maybe it should be fixed in Nacos.
> > 
> > 
> > yeap, I have fix it in [alibaba/spring-cloud-alibaba#3545](https://github.com/alibaba/spring-cloud-alibaba/pull/3545)
> 
> good job , waiting the bug fixed version of nacos used in shenyu

ShenYu has fixed it
